addons/udes_stock_package_move/tests/test_change_package_location.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from odoo.exceptions import UserError
+from addons.udes_stock.tests import common
+
+
+class TestChangePackageLocation(common.BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestChangePackageLocation, cls).setUpClass()
+
+        Package = cls.env["stock.quant.package"]
+
+        cls.empty_package = Package.create({})
+        cls.package_1 = cls.create_package(cls.test_stock_location_01)
+        cls.package_2 = cls.create_package(cls.test_stock_location_01)
+        cls.package_3 = cls.create_package(cls.test_stock_location_01)
+        cls.package_4 = cls.create_package(cls.test_stock_location_02)
+        cls.packages = cls.package_1 | cls.package_2 | cls.package_3 | cls.package_4
+
+    @classmethod
+    def create_package(cls, location):
```

Comment:
Note: `create_package` already exists in `common.BaseUDES`. Shall we rename this one ?